searcher.proto

```diff
@@ -59,8 +61,14 @@ message GetTipAccountsResponse {
 }
 
 service SearcherService {
-  // RPC endpoint to subscribe to pending transactions. Clients can provide a list of base58 encoded accounts.
-  // Any transactions that write-lock the provided accounts will be streamed to the searcher.
+  // Searchers can invoke this endpoint to subscribe to their respective bundle results.
+  // A success result would indicate the bundle won its state auction and was submitted to the validator.
+  rpc SubscribeBundleResults (google.protobuf.Empty) returns (stream bundle_result.BundleResult) {}
```

Comment:
nit: wdyt about making empty request object? then if we wanna add fields, can start adding optional fields vs. changing datatype

Reply:
wfm